### PR TITLE
Update size answer in FAQ

### PR DESCRIPTION
**Pull request by sethladd**
Fixes https://github.com/flutter/flutter/issues/9659

### CODE REVIEW

**Comment by sethladd** [comment]
cc @Hixie

**Comment by sethladd** [comment]
I got the results from:

```
Archive:  app-release.apk
 Length   Method    Size  Cmpr    Date    Time   CRC-32   Name
--------  ------  ------- ---- ---------- ----- --------  ----
    2284  Defl:N      850  63% 00-00-1980 00:00 ae7cf045  AndroidManifest.xml
     765  Defl:N      596  22% 00-00-1980 00:00 742b6e3f  META-INF/CERT.RSA
    1309  Defl:N      641  51% 00-00-1980 00:00 d4106dc8  META-INF/CERT.SF
    1266  Defl:N      615  51% 00-00-1980 00:00 fd25858a  META-INF/MANIFEST.MF
   57817  Defl:N    56003   3% 00-00-1980 00:00 d43a9ab8  assets/app.flx
 6266928  Defl:N  2200895  65% 12-31-1969 16:00 98b9d336  assets/icudtl.dat
 2094328  Defl:N   835276  60% 00-00-1980 00:00 5606571d  assets/isolate_snapshot_data
 2469376  Defl:N   466806  81% 00-00-1980 00:00 d0bd85f2  assets/isolate_snapshot_instr
   21936  Defl:N    11814  46% 00-00-1980 00:00 a34c0d2d  assets/vm_snapshot_data
   10160  Defl:N     2178  79% 00-00-1980 00:00 152498c2  assets/vm_snapshot_instr
  108344  Defl:N    41513  62% 00-00-1980 00:00 10e86dfe  classes.dex
 6701224  Defl:N  3389738  49% 12-31-1969 16:00 a55081c5  lib/armeabi-v7a/libflutter.so
     507  Stored      507   0% 00-00-1980 00:00 05af2edb  res/mipmap-hdpi-v4/ic_launcher.png
     404  Stored      404   0% 00-00-1980 00:00 60b53227  res/mipmap-mdpi-v4/ic_launcher.png
     684  Stored      684   0% 00-00-1980 00:00 3df0e9b9  res/mipmap-xhdpi-v4/ic_launcher.png
     995  Stored      995   0% 00-00-1980 00:00 247b1622  res/mipmap-xxhdpi-v4/ic_launcher.png
    1408  Stored     1408   0% 00-00-1980 00:00 3f49dcf6  res/mipmap-xxxhdpi-v4/ic_launcher.png
    1156  Stored     1156   0% 00-00-1980 00:00 49d61f62  resources.arsc
```

File: faq.md [hunk 1]
```diff
@@ -256,11 +256,19 @@ or directly) dart:mirrors or dart:html.
...
-2.5MB of ICU data.
+In June 2017, we measured the size of a minimal Flutter app
+(no Material Design, just a single `Center` widget, built with
+`flutter build apk`), bundled and
+compressed as a release APK, to be approximately 6.7MB.
+
+For this simple app,
+the core engine is approximately 3.3MB (compressed),
+the framework + app code is approximately 1.25MB bytes (compressed),
+necessary Java code (classes.dex) is 40k (compressed),
```
Comment (Hixie) [inline faq.md]: Why do you mention classes.dex but not the flx?

Comment (sethladd) [inline faq.md]: There's nothing in the .flx for the "necessary Java code"

Comment (Hixie) [inline faq.md]: Also, holy cow, we went from 400KB to 1.25MB? What did we add!

Comment (sethladd) [inline faq.md]: oh, I see... because the .flx is empty for this simple app.

Comment (Hixie) [inline faq.md]: The FLX is 56KB compresed here, the classes.dex is 42KB. It seems like if you mention one you should mention the other.

Comment (sethladd) [inline faq.md]: We think we took the previous measurement so long ago, we have a fundamentally different way of compiling and bundling. The last measurement was 2015.

Comment (sethladd) [inline faq.md]: Ah, the .flx contains the LICENSE, which is a constant cost. So, yes, I should include it here. The LICENSE takes 57k compressed :)

**Comment by Hixie** [comment]
LGTM

File: faq.md (hunk 1 above)
Comment (Hixie) [inline faq.md]: Wow, that is some hot compression there. The original file is over 800KB!

**Pull request merged**